oeps/architectural-decisions/oep-0065/decisions/0002-frontend-app-migrations.rst

```diff
@@ -0,0 +1,120 @@
+OEP-65 Frontend App Migrations
+##############################
+
+Status
+******
+
+Accepted
+
+Summary
+*******
+
+This ADR describes a migration path and simplified repository organization for the frontend-app-* repositories that is compatible with the shell provided by frontend-base as well as module federation and frontend projects.
+
+Context
+*******
+
+OEP-65 proposes adopting webpack module federation for Open edX micro-frontends (MFEs) as a means to enable *runtime module loading* and *shared dependencies* in the Open edX frontend.  The OEP lays out a series of changes necessary to enable these capabilities, which it refers to as building a "reference implementation" of runtime module loading and shared dependencies.
+
+This reference implementation is effectively a new underlying architecture for our frontend. This ADR refers to this new architecture as the "module" architecture, as opposed to the historical "micro-frontend" architecture that has existed prior to OEP-65.
+
+As part of this paradigm shift, our frontend-app-* repositories (MFEs) will need to be migrated to work with frontend-base.  Of particular note, this will require the repositories to:
+
+- Adopt a new set of build/development CLI helpers
+- Use the shell to provide the header, footer, and runtime initialization code, amongst other things.
+- Organize their code into loosely-coupled top-level components, which we'll call *modules*.
+
+As we adopt frontend-base, the libraries it replaces will undergo their own deprecation processes, which will need to coordinate with the migration of micro-frontends included in Open edX releases.  After that deprecation, the micro-frontend architecture will cease to be supported.
+
+Decision
+********
+
+Each of our frontend-app-* repositories will migrate from being an independent "micro-frontend application" to being a library of modules that can be loaded into a common Shell, deployed as a Site.  We will document the migration process in detail.  At a high level:
+
+- They will be deployable via a variety of methods, all of which will use the Shell.
+- They will be deployable together or independently.
+- They will no longer contain .env or env.config files for any environment, including Devstack and Tutor.
+- They will cease to use the following libraries in favor of frontend-base:
+  - @openedx/frontend-build
```

Comment:
```suggestion
- They will cease to use the following libraries in favor of frontend-base:

  - @openedx/frontend-build
```

drive-by comment -- in rST (unlike in MD) you need a blank line before a sublist, otherwise it will render all smashed together on one line. there are a couple other instances below the need the blank line.